docs/source/contributor-guide/development.md

```diff
@@ -85,6 +85,32 @@ mvn test -Dsuites="org.apache.comet.CometCastSuite valid" -Dskip.surefire.tests=
 
 Other options for selecting specific suites are described in the [ScalaTest Maven Plugin documentation](https://www.scalatest.org/user_guide/using_the_scalatest_maven_plugin)
 
+## Plan Stability Testing
+
+Comet has a plan stability testing framework that can be used to test the stability of the query plans generated by Comet.
+The plan stability testing framework is located in the `spark` module and can be run using the following command:
+
+```
+mvn -pl spark -Dsuites="org.apache.spark.sql.comet.CometTPCDSV1_4_PlanStabilitySuite" test
```

Comment:
I think we should recommend to use `./mvnw`

Reply:
Okay